backend/src/ml_space_lambda/enums.py

```diff
@@ -54,6 +54,43 @@ class ServiceType(str, Enum):
     TRANSFORM_JOB = "transformJob"
 
 
+class IAMStatementProperty(str, Enum):
+    ACTION = "Action"
+    EFFECT = "Effect"
+    RESOURCE = "Resource"
+    CONDITION = "Condition"
+
+
+class IAMEffect(str, Enum):
+    DENY = "Deny"
+    ALLOW = "Allow"
+
+
+class EnvVariable(str, Enum):
```

Comment:
If you're going to add this you should [update this](https://github.com/awslabs/mlspace/blob/main/backend/src/ml_space_lambda/utils/mlspace_config.py) at a minimum.